src/evaluate/evaluator/utils.py

```diff
@@ -16,3 +16,13 @@ def __getitem__(self, i):
 
     def __iter__(self):
         return (self.dataset[i][self.key] for i in range(len(self)))
+
+
+def choose_split(data):
+    available_splits = get_dataset_split_names(data)
+    if "test" in available_splits:
+        return "test"
+    elif "validation" in available_splits:
+        return "validation"
+    else:
+        return "train"
```

Comment:
We should check if `train` is actually present in `available_splits` - if not, I think it would make sense to just return the smallest split of the available ones regardless of the name? Perhaps @lvwerra could weigh in here

Reply:
Have addressed in above comment :)